src/plasma/plasma_manager.h

```diff
@@ -1,6 +1,8 @@
 #ifndef PLASMA_MANAGER_H
 #define PLASMA_MANAGER_H
 
+#include "protocol.h"
```

Comment:
In https://github.com/apache/arrow/pull/2282/files , this file is removed from the installation list, so this file is not available now.

Reply:
oh yeah, it's a new file with forward declarations that I forgot to add :)